package day15Exception;

public class Test {
    //try blogu mutlaka catch blogu ile kullanilmalidir
    //True/False
    //false
    //o finally blogu mutlaka calisir True/False
    //true
    //o Bir try blogu ile birden fazla catch blogu calistirilabilir
    //True/False
    //true
    //o Birden fazla catch blok varsa, child olan once
    //yazilmalidir. True/False
    //false
    //o FileNotFoundException nedir?
    // yeni bir örneğini belirtilen hata iletisiyle ve bulunamayan dosya adıyla başlatır.
    //aranan degeri bulamadıgını söyleyen exception dur
    //o IOException nedir?
    //akışlar, dosyalar ve dizinler kullanılarak bilgilere erişilirken oluşan özel durumlar için temel sınıftır.
}
